common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java

```diff
@@ -62,6 +70,12 @@ public static void setSystemProperties(String spanEndpoint,
     if (logLevel != null) {
       System.setProperty("io.opentelemetry.auto.slf4j.simpleLogger.defaultLogLevel", logLevel);
     }
+
+    String otelResourceAttributes = "service.name=" + serviceName;
```

Comment:
I think we want to keep any previous value the user specified here, and append our values here. I.e.:

```java
if (existingResourceAttributes != null) {
  // append our service.name + service.version
} else {
  // Do as you do now.
}
```